web/modules/custom/ai_screening_project_track/src/Form/ThresholdsForm.php

```diff
@@ -100,7 +102,31 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
     }
 
     $form['#validate'][] = $this->validateThresholds(...);
-    $form['actions'] = [
+
+    $form['form_footer'] = [
+      '#type' => 'container',
+      '#attributes' => ['class' => ['flex', 'justify-between']],
+    ];
+
+    $url = Url::fromUri($_SERVER['HTTP_REFERER']);
```

Comment:
You should not use `$_SERVER`, but get the referrer from the request (see https://www.drupal.org/node/2150267 and https://drupal.stackexchange.com/a/257569 for examples).

You should also check that the referrer is actually set before generating the URL.

Reply:
Fixed